contracts/secret-storage/TeeManager.sol

```diff
@@ -568,12 +570,12 @@ contract TeeManager is
         uint256 remainingStakeAmount = stakeAmount;
         uint256 iterations = _missedEpochsCount / MAX_EPOCHS_PER_ITERATION;
         for (uint256 i = 0; i < iterations; i++) {
-            remainingStakeAmount = remainingStakeAmount * ((SLASH_MAX_BIPS - SLASH_PERCENT_IN_BIPS) ** MAX_EPOCHS_PER_ITERATION) / (SLASH_MAX_BIPS ** MAX_EPOCHS_PER_ITERATION);
+            remainingStakeAmount = remainingStakeAmount.mulDiv(((SLASH_MAX_BIPS - SLASH_PERCENT_IN_BIPS) ** MAX_EPOCHS_PER_ITERATION), (SLASH_MAX_BIPS ** MAX_EPOCHS_PER_ITERATION));
```

Comment:
with this calculation, we don't have to worry about overflowing because of multiplication, it can max 2^256 - 1, right? Test it out for max numbers.

Reply:
yes working perfectly fine, added UT